src/Components/Browser.JS/src/GlobalExports.ts

```diff
@@ -1,13 +1,16 @@
-import { platform } from './Environment';
 import { navigateTo, internalFunctions as uriHelperInternalFunctions } from './Services/UriHelper';
 import { internalFunctions as httpInternalFunctions } from './Services/Http';
 import { attachRootComponentToElement } from './Rendering/Renderer';
-import { Pointer } from './Platform/Platform';
+
+// Accept the following callbacks that need to be set before we boot.
+const existing = window['Blazor'];
```

Comment:
The idiomatic way to do this in JS is window['Blazor'] = window['Blazor'] || {};